org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/validation/XbaseUIValidator.java

```diff
@@ -84,6 +86,19 @@ public void checkRestrictedType(XConstructorCall constructorCall) {
 		checkRestrictedType(constructorCall, XbasePackage.Literals.XCONSTRUCTOR_CALL__CONSTRUCTOR, declaringType);
 	}
 
+	@Check
+	public void checkRestrictedType(XMemberFeatureCall featureCall) {
```

Comment:
Should probably be defined for XAbstractMemberFeatureCall, otherwise we'll only have the warning on the import but not on the reference.